Utilities/FrameworkResources/exports.exp

```diff
@@ -18,6 +18,7 @@ _HCHttpCallGetId
 _HCHttpCallSetTracing
 _HCHttpCallGetRequestUrl
 _HCHttpCallRequestEnableGzipCompression
+_HCHttpCallSetCompressResponse
```

Comment:
Not sure if the same comment about the .def files applies for other platforms, but might be worth confirming